### ISSUE TITLE
add android tv support

### ISSUE THREAD

**Issue by ClrsDream** [vpnhood/VpnHood#99]

**Comment by pourmirzai** [comment]
> I thought it was working there. Have you tried it on Android TV? Would you let me know the main issue?

Google Play doesn't allow you to install it directly on Android TV. I tried to share the app via my phone and the installation was unsuccessful as well.
However, I downloaded the APK file version 2.4.307 from APKpure and it's working on my Mibox S.

**Comment by trudyhood** [comment]
@pourmirzai We haven't officially supported it but tried to make it compatible. Still couldn't find to find an easy way to transfer accessKey from mobile to Android TV without any account. You can't count on Public Servers.

**Comment by pourmirzai** [comment]
@trudyhood I'm using a private server and it works fine. Thanks.

**Comment by trudyhood** [comment]
> @trudyhood I'm using a private server and it works fine. Thanks.

Nice, Would you let me know how did you manage to copy the accessKey to your TV?

**Comment by pourmirzai** [comment]
@trudyhood 
Of course! I have paired my phone with the anddroid box as a remote control via Google Home app. So I navigate with my phone and use my phone's keyboard and clipboard to paste the texts in the fields.
